spec.bs

```diff
@@ -5975,17 +6064,24 @@ enum KAnonStatus { "passedAndEnforced", "passedNotEnforced", "belowThreshold", "
   <dd>True if the interest group owner was the only bidder that made bids with the second highest
     score
   <dt>{{ReportWinBrowserSignals/buyerReportingId}}
-  <dd>Set if the winning ad had a [=interest group ad/buyer reporting ID=] but not a
-    [=interest group ad/buyer and seller reporting ID=] set in its listing in the interest group,
-    and that value was [=query reporting ID k-anonymity count|jointly k-anonymous=] combined with
-    interest group owner, bidding script URL, and [=ad creative=] URL.
+  <dd>There are two scenerios in which this can be set:
+    * Set if the winning ad had a [=interest group ad/buyer reporting ID=] but not a
+      [=interest group ad/buyer and seller reporting ID=] set in its listing in the interest group,
+      and the winning bid did not have [=generated bid/selected buyer and seller reporting ID=] set,
+      and that value was [=query reporting ID k-anonymity count|jointly k-anonymous=] combined with
+      interest group owner, bidding script URL, [=ad creative=] URL, and null.
+    * Set if the wining bid had a [=generated bid/selected buyer and seller reporting ID=] and the winning ad had a
+      [=interest group ad/buyer reporting ID=] set in its listing in the interest group, and that value was
+      [=query reporting ID k-anonymity count|jointly k-anonymous=] combined with
+      [=interest group ad/buyer and seller reporting ID=], interest group owner, bidding 
+      script URL, [=ad creative=] URL, and [=generated bid/selected buyer and seller reporting ID=],.
 
   <dt>{{ReportWinBrowserSignals/interestGroupName}}
-  <dd>Only set if the tuple of interest group owner, name, bidding script URL and [=ad creative=] URL
-
+  <dd>Only set if the tuple of interest group owner, name, bidding script URL, [=ad creative=] URL, and null
     were [=query reporting ID k-anonymity count|jointly k-anonymous=], and the

```

Comment:
this whole sentence is quite hard to understand now. Can we rewrite it a little bit to make the logic clearer?

Reply:
just changed it let me know if that's clearer logic!